src/modules/45-dashboards/components/AidaDrawer/__tests__/AidaDrawer.test.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { act, fireEvent, render, RenderResult, screen } from '@testing-library/react'
+import { TestWrapper } from '@common/utils/testUtils'
+import AidaDrawer, { AidaDrawerProps } from '..'
+
+const renderComponent = (props: AidaDrawerProps): RenderResult =>
+  render(
+    <TestWrapper>
+      <AidaDrawer {...props} />
+    </TestWrapper>
+  )
+
+describe('AidaDrawer', () => {
+  const testContentText = 'Test Content'
+  const testComponent = <div>{testContentText}</div>
+
+  test('it should not display content within drawer when closed', async () => {
+    renderComponent({ children: testComponent, isOpen: false, setIsOpen: jest.fn() })
+
+    expect(screen.queryByText('dashboards.aida.assist')).toBeNull()
+    expect(screen.queryByText(testContentText)).toBeNull()
```

Comment:
Try to use `not.toBeInTheDocument` instead to better convey what you're trying to prove.
```suggestion
    expect(screen.queryByText('dashboards.aida.assist')).not.toBeInTheDocument()
    expect(screen.queryByText(testContentText)).not.toBeInTheDocument()
```